src/NuGet.Clients/NuGet.CommandLine/NuGet.CommandLine.csproj

```diff
@@ -120,14 +120,6 @@
     <Exec Command="$(IlmergeCommand)" ContinueOnError="false" />
   </Target>
 
-  <Target Name="NuGetLinkVSSDKEmbeddableAssemblies" AfterTargets="ResolveReferences" BeforeTargets="FindReferenceAssembliesForReferences">
-    <ItemGroup>
-      <ReferencePath Condition="'%(FileName)' == 'Microsoft.VisualStudio.Setup.Configuration.Interop'">
-        <EmbedInteropTypes>true</EmbedInteropTypes>
```

Comment:
Sorry I completely forgot to call out why this is okay.  I updated us to a newer version of [Microsoft.VisualStudio.Setup.Configuration.Interop](https://github.com/NuGet/NuGet.Client/pull/4931/files#diff-5baf5f9e448ad54ab25a091adee0da05d4d228481c9200518fcb1b53a65d4156R56) because newer versions of the package injects this logic for us:

![image](https://user-images.githubusercontent.com/17556515/202872018-010f171e-0e09-419c-9b73-3d52c7a169b1.png)

I had remembered to double check but forgot to post anything here, thanks for keeping me honest 😄 

![image](https://user-images.githubusercontent.com/17556515/202872066-f7ad3788-e121-4264-8f43-42a5f4c427a2.png)